src/data/Thread.js

```diff
@@ -90,15 +90,16 @@ class Thread {
       let firstMessage;
 
       const textContent = typeof content === "string" ? content : content.content;
+      const contentObj = typeof content === "string" ? {} : content;
       if (textContent) {
         // Text content is included, chunk it and send it as individual messages.
         // Files (attachments) are only sent with the last message.
         const chunks = utils.chunkMessageLines(textContent);
         for (const [i, chunk] of chunks.entries()) {
           // Only send embeds, files, etc. with the last message
           const msg = (i === chunks.length - 1)
-            ? await bot.createMessage(this.channel_id, chunk, file)
-            : await bot.createMessage(this.channel_id, chunk);
+            ? await bot.createMessage(this.channel_id, { ...contentObj, content: chunk }, file)
+            : await bot.createMessage(this.channel_id, { ...contentObj, content: chunk });
```

Comment:
I just realized this would now send any embeds with each message. I think for line 102 (i.e. all but the last chunk), let's do `allowedMentions: contentObj.allowedMentions` without the spread, and keep the spread for line 101.

Reply:
Hmm, i dont know if that is a good idea. I spread on purpose in case any other option ever becomes relevant (or a new one gets added).
I would rather go `{ ...contentObj, content: chunk,  embed: null}` if thats fine.

Reply:
That's a good point. Need to double-check if `embed: null` will post correctly, but that should work.

Reply:
Fixed with e2bb333